setup_toolchains.py

```diff
@@ -42,9 +49,9 @@
 
 def create_wrappers(toolchain_dir, root_dir):
   for toolchain in _AVAILABLE_TOOLCHAINS:
-    host_os_name = '' if toolchain['host_os'] == 'linux' else "-" + toolchain['host_os']
+    host_os_name = '' if toolchain['host_os'] == 'linux' else toolchain['host_os'] + "-"
     name = ('arm-gnu-toolchain-' +
-            f'{toolchain["version"]}-{toolchain["host_arch"]}{host_os_name}-{toolchain["target"]}')
+            f'{toolchain["version"]}-{host_os_name}{toolchain["host_arch"]}-{toolchain["target"]}')
```

Comment:
Was this a latent bug?

Reply:
I think so but maybe you just intended to not include the host os name. cc @ograff